src/utils/files.ts

```diff
@@ -1,4 +1,6 @@
 export const serializeDateForFileName = (date: Date) =>
-  `${date.getUTCFullYear()}-${
-    date.getUTCMonth() + 1
-  }-${date.getUTCDate()}-${date.getUTCHours()}-${date.getUTCMinutes()}`;
+  `${date.getUTCFullYear()}-`
```

Comment:
There is a `formatDateForFileName` function in `src/modules/migrations/utils/migrationUtils.ts` that has the same implementation. Remove it and replace its calls to this function instead.

Reply:
Oh, I noticed it also has seconds in the result. Maybe we can parametrize this?

Reply:
I didn't have anything in mind. 😄 But it's true that the single boolean parameter is rather random. Since we don't want formats like YYYY-DD, we can instead have a parameter specifying the maximum detail (e.g. "year" -> YYYY, "minute" -> YYYY-DD-hh-mm").

Reply:
I pushed a suggested refactor of the function. Feel free to remove/squash it.